ursa/graph/graph.py

```diff
@@ -234,3 +235,13 @@ def getattr(self, item):
         @return: The attribute requested.
         """
         return getattr(self, item)
+
+    def clean_old_rows(self, graph_id, versions_to_store):
```

Comment:
This needs proper docstring commenting (see Slack message for reference)

Reply:
Resolved.